src/convertToObject.js

```diff
@@ -6,7 +6,40 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
+  const cleanedSourceString = sourceString.replace(/\t+/, ' ').trim();
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. The `split` method does not mutate the original string, so you can chain the `split` method directly after the `replace` method.